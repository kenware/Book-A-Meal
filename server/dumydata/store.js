const store = {
 meals:[
  {
    id: 1,
    name: 'Rice and beans',
    price: 500,
    description: 'very delicious'
  },
  {
    id: 2,
    name: 'Abacha and Fish',
    price: 1000,
    description: 'very Sumptious'
  },
  {
    id: 3,
    name: 'Amala and Ewedu',
    price: 500,
    description: 'very delicious'
  }
],
orders:  [
  {
    id: 1,
    userId: 4,
    status: 'pending',
    quantity: 2,
    totalPrice: 1000,
    meal: {
      id: 1,
      name: 'Rice and beans',
      price: 500,
      description: 'very delicious'
    }
  }
 ],
 menus:[
  {
    id: 1,
    title: 'today menu',
    menuDate: '2018-04-23',
    meals: [{
      id: 1,
      name: 'Rice and beans',
      price: 500,
      description: 'very delicious'
    }]    
  }
]


};
export default store;
