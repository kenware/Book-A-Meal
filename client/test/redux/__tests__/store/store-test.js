import store from '../../../../redux/store';

describe('allOrderReducer reducer', () => {
  it('should return the initial state', () => {
    expect(store()).toMatchSnapshot();
  });
});
