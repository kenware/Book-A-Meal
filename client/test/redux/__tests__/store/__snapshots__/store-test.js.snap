// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allOrderReducer reducer should return the initial state 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
